tools/editor/project_manager.cpp

```diff
@@ -871,8 +871,8 @@ ProjectManager::ProjectManager() {
 
 	HBoxContainer *top_hb = memnew( HBoxContainer);
 	vb->add_child(top_hb);
-	TextureFrame *logo = memnew( TextureFrame );
-	logo->set_texture(theme->get_icon("LogoSmall","EditorIcons"));
+	//TextureFrame *logo = memnew( TextureFrame );
+	//logo->set_texture(theme->get_icon("LogoSmall","EditorIcons"));
```

Comment:
If the code is no longer needed, then remove it. We are using VCS so there is no problem searching back in time.